apps/common-app/src/examples/RuntimeTests/tests/animations/withTiming/arrays.test.tsx

```diff
@@ -102,6 +101,7 @@ describe('withTiming animation of ARRAY', () => {
       );
       const components = Object.values(COMPONENT_REF).map(refName => getTestComponent(refName));
 
+      // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Why disable that? 👀

Reply:
Because syntax
```tsx
array.foreach(async () => {})
```
is not allowed. I'll replace it with `for (const X of array)` loop.